string autosequence_name
---
bool found
flyer_controller/Autosequence autosequence